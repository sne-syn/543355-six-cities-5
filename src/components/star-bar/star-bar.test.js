import React from 'react';
import renderer from 'react-test-renderer';
import StarBar from './star-bar';

test(`Render StarBar`, () => {
  const tree = renderer
    .create(
        <StarBar
          rating={4}
          containerClassName={`place-card`}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
