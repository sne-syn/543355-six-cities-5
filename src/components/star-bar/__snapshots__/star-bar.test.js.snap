// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render StarBar 1`] = `
<div
  className="place-card__rating rating"
>
  <div
    className="place-card__stars rating__stars"
  >
    <span
      style={
        Object {
          "width": "80%",
        }
      }
    />
    <span
      className="visually-hidden"
    >
      Rating
    </span>
  </div>
  
</div>
`;
